packages/utils/perf/score-report/optimized0.ts

```diff
@@ -17,16 +18,11 @@ function groupRefToScore(audits: EnrichedAuditReport[]) {
   };
 }
 
-function categoryRefToScore(
-  audits: EnrichedAuditReport[],
-  groups: EnrichedScoredGroup[],
-) {
+function categoryRefToScore(audits: AuditReport[], groups: ScoredGroup[]) {
   return (ref: CategoryRef) => {
     switch (ref.type) {
       case 'audit':
-        const audit = audits.find(
-          a => a.slug === ref.slug && a.plugin === ref.plugin,
-        );
+        const audit = audits.find(a => a.slug === ref.slug);
```

Comment:
Why was this changed? The audit `slug` is not unique, you have to check the `plugin` slug as well.

(Same goes for `case 'group'`.)

Reply:
This is in the `perf` folder. So here I had two options - either just ignore the errors or do best effort.
Since this benchmark uses the old type, I would otherwise have to refactor this function...

Reply:
Ah, I see. Then f*** it :laughing: